rfcs/0145-doc-strings.md

```diff
@@ -119,19 +117,157 @@ In the following we give a comprehensive overview to our decision that we've mad
 
 Adopting CommonMark as the content for all doc-comments brings the benefits of widely accepted and understood documentation format in tech projects, while maintaining profitability and consistency within the Nix ecosystem by aligning with existing [NixOS/RFC-72](https://github.com/NixOS/rfcs/blob/master/rfcs/0072-commonmark-docs.md).
 
-## `/**` starts a doc-comment
+The content indentation inherits the indentation logic from nixs' multiline strings `'' ''` allowing any arbitrary indentation the user might prefer.
+
+## `/** */` is the doc-comment format
 
 The decision to use /** to start a doc-comment ensures a unique distinction from regular comments while still allowing seamless writing without IDE or editor support. This choice not only provides the best developer experience but also minimizes the need for additional tooling overhead.
 
 ## Placement
 
-> TODO: Finding a universal approach for describing the appropriate placement that effectively links doc-comments with corresponding code elements.
+- Documentable nodes are `lambda value` and `attribute name` only. (This might be extended in the future)
 
+- Doc-comments are placed before the documentable node. Only `WHITESPACES` are allowed in between.
+    - `WHITESPACES` are: `[\n \r ' ' \t]`, singleline- and multiline-comments (that are not doc-comments). 
 
+Accurate and usable Lambda documentation reqires some more details to be specified:
 
+- **If a lambda is assigned to an `attribute name`, then the `attribute name`-documentation is also the `lambda value` documentation**.
```

Comment:
I'd like to suggest not using the word `assign`, because there is actually no "assignment" happens in the parser, it is something similar to a json field, not an assignment. For C++ nix there is no this term actually, we just treat this token as a part of an attribute, which is a part of `binds` (NTerm).